analytics/views.py

```diff
@@ -561,6 +561,8 @@ def realm_summary_table(realm_minutes: Dict[str, float]) -> str:
                 WHERE
                     property = 'realm_active_humans::day'
                     AND end_time > now() - interval '25 hours'
+                ORDER BY id DESC
+                LIMIT 1
```

Comment:
Is that going to limit to 1 per realm, or 1 overall?

Reply:
Maybe instead of the `interval 25 hours` we want to use FillState to figure out what the latest data is?